.circleci/config.yml

```diff
@@ -317,17 +329,24 @@ workflows:
       # E2E: PWA
       - cypress/run:
           name: 'E2E: PWA'
-          executor: cypress/browsers-chrome76
+          executor: with-chrome
```

Comment:
This should probably be a specific version? Actually, probably a good thing to always test using latest browser/vendor in this instance.

Reply:
with-chrome is the executor, it has a specific version in it at the top of the file